nautobot_firewall_models/utils/__init__.py

```diff
@@ -2,17 +2,84 @@
 
 import json
 
-from django.conf import settings
+from django.apps import apps as django_apps
 from django.utils.module_loading import import_string
 from nautobot.core.models.utils import serialize_object_v2
-from nautobot.extras.models import Status
+
+try:
+    from nautobot.extras.management import STATUS_COLOR_MAP, STATUS_DESCRIPTION_MAP
+except ImportError:  # Nautobot version < v2.2.0
+    from nautobot.extras.management import COLOR_MAP as STATUS_COLOR_MAP
+    from nautobot.extras.management import DESCRIPTION_MAP as STATUS_DESCRIPTION_MAP
 from rest_framework.renderers import JSONRenderer
 
+from nautobot_firewall_models.constants import PLUGIN_CFG
+
+
+def _create_status(status_name, apps=django_apps):
+    """Create a status with the given name, using nautobot default description and color if applicable."""
+    Status = apps.get_model("extras.Status")  # pylint: disable=invalid-name
+    defaults = {"description": STATUS_DESCRIPTION_MAP.get(status_name, "")}
+    if status_name in STATUS_COLOR_MAP:
+        defaults["color"] = STATUS_COLOR_MAP[status_name]
+    status, _ = Status.objects.get_or_create(name=status_name, defaults=defaults)
```

Comment:
What happens if the status "active" exists and the default "Active" is there?

Reply:
It's case sensitive so it will choose whichever status matches.